Mathlib/Algebra/Group/Units/Defs.lean

```diff
@@ -357,10 +357,9 @@ variable {M : Type*} {N : Type*}
 /-- An element `a : M` of a `Monoid` is a unit if it has a two-sided inverse.
 The actual definition says that `a` is equal to some `u : Mˣ`, where
 `Mˣ` is a bundled version of `IsUnit`. -/
-@[to_additive
-      "An element `a : M` of an `AddMonoid` is an `AddUnit` if it has a two-sided additive inverse.
-      The actual definition says that `a` is equal to some `u : AddUnits M`,
-      where `AddUnits M` is a bundled version of `IsAddUnit`."]
```

Comment:
Was there consensus about `to_additive` docstrings?

Reply:
I asked specifically [on zulip]([#mathlib4 > Style :bicycle: : indenting second lines in doc-strings @ 💬](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Style.20.3Abicycle.3A.20.3A.20indenting.20second.20lines.20in.20doc-strings/near/506201449))

Reply:
For the record: I think the idea to change to_additive's syntax is a fine one, but far wider-reaching than this change. Let's not block this PR on it.